src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms linear;
```

```suggestion
    transition: transform 300ms linear, color 300ms linear;
```
В transition потрібно вказувати до яких саме властивостей використовуєтья transition, по дефолту стоїть all, але казали щоби не робили так, а чітко вказували)

---

Fix it

---

Дякую за пояснення, бо я спочатку не поняв що саме не корректно.